### PR TITLE
Refactor actions

### PR DESCRIPTION
**Pull request by Gman1999**

### CODE REVIEW

**Comment by mindedj** [comment]
I've just merged auto_recommendation into master, please rebase your branch first so it's up-to-date.

File: actions/actions.py [hunk 1]
```diff
@@ -15,17 +15,17 @@ def get_available_appointments():
...
-    now = datetime.now()
+    now = datetime.now()  # Get the current date
 
-    with open('test_data.json', 'r') as f:
+    with open('test_data.json', 'r') as f:  # Opening test_data
         data = json.load(f)
 
     res = []
-    for day in data:
+    for day in data:                                                      # Parsing the test_data into a dictionary
```
Comment (mindedj) [inline actions/actions.py:24]: Please leave only 1 space between code and comment

File: actions/actions.py [hunk 2]
```diff
@@ -35,14 +35,14 @@ def get_available_appointments():
     return res
 
 
-def get_date_text(dt):
+def get_date_text(dt):                                     # Converting dates into text
```
Comment (mindedj) [inline actions/actions.py:38]: For function description please use `"""docstring"""`.

File: actions/actions.py [hunk 3]
```diff
@@ -35,14 +35,14 @@ def get_available_appointments():
...
     dt = datetime.combine(dt, datetime.min.time())
     candidates = datetime2text(dt, time_precision=2)
 
     return f"{candidates['dates'][0]}"
 
 
-def get_time_text(dt, add_suffix=False):
+def get_time_text(dt, add_suffix=False):                # Converting text into dates
```
Comment (mindedj) [inline actions/actions.py:45]: I think it actually converts datetime to text. `get_time_text(datetime(2021,3,25,7,0))` returns `reggel 7 óra`

**Review by mindedj — COMMENTED** [review]
Good work, please resolve the issues I mentioned.

File: actions/actions.py (hunk 3 above)
Comment (Gman1999) [reply actions/actions.py]: I made the changes you asked for.

**Comment by Gman1999** [comment]
> Good work, please resolve the issues I mentioned.

I don't know how to rebase a branch correctly

File: actions/actions.py [hunk 4]
```diff
@@ -36,13 +39,25 @@ def get_available_appointments():
...
+
     dt = datetime.combine(dt, datetime.min.time())
     candidates = datetime2text(dt, time_precision=2)
 
     return f"{candidates['dates'][0]}"
 
 
 def get_time_text(dt, add_suffix=False):
+    """
+<<<<<<< HEAD
```
Comment (mindedj) [inline actions/actions.py:54]: You have conflicting lines in `get_time_text` and `get_common_intervals` for some reason.

Comment (Gman1999) [reply actions/actions.py]: Oops, I remove them right now.

**Review by mindedj — APPROVED** [review]
Good job, you can merge your branch into master.

**Pull request merged**